meson_options.txt

```diff
@@ -43,4 +43,12 @@ option('efi-libdir', type : 'string', description : 'path to the EFI lib directo
 option('efi-ldsdir', type : 'string', description : 'path to the EFI lds directory')
 option('efi-includedir', type : 'string', value : '/usr/include/efi', description : 'path to the EFI header directory')
 option('efi_os_dir', type: 'string', description : 'the name of OS directory in ESP')
+option('efi_sbat_spec_version', type : 'integer', value : 1, description : 'SBAT specification version')
```

Comment:
spec is starting at 1?  I thought in @martinezjavier PR it was start 0

Reply:
We went back and forth on this; I think it will indeed start at 1.

Reply:
I was also thinking it would be 0... I don't think we've landed on an official answer yet.

Reply:
From a 40,000ft view, a specification version 0 doesn't inspire confidence. It certainly says "_prerelease, not ready_" to me.

Reply:
I think that agree with @hughsie on this. Specially for the `sbat` entry.